logstash-core/lib/logstash/pipeline.rb

```diff
@@ -180,7 +182,9 @@ def start
         @finished_run.make_true
       rescue => e
         close
-        @logger.error("Pipeline aborted due to error", default_logging_keys(:exception => e, :backtrace => e.backtrace))
+        pipeline_log_params = default_logging_keys(:exception => e, :backtrace => e.backtrace)
```

Comment:
Nit again, no need for extra assignment, add the extra field to the `default_logging_keys` method call